README.md

```diff
@@ -1,10 +1,17 @@
 # Market-analysis
 ## Group project for Code 401 - Python
+##### Our live site on heroku:
 ### Contributors:
 * David Smith
-* David Banks 
+* David Banks
 * Tatiana Weaver
 
-### This is a Python web app, implemented in Pyramid. 
-### The app is designed to display change of the stock value over time for particular companies.
+### This is a Python web application, implemented in Pyramid.
+#### This app is designed to keep track of stocks that can be added to your portfolio.  We maintain our user information and associations with a database, and use the MarkItOnDemand API to get our information about stocks in the user's portfolio.  We feature user accounts with two different levels of permissions for views.  Test modules have been created in order to ensure proper functionality of the website.  To view the portfolio, we display a graph showing percent changes, and we also have a graph showing price data for each individual stock.
```

Comment:
Do you really want this to be one H4 tag?  It doesn't seem like this should be a headline, but rather a paragraph.